feat(posts): add devfact 14 (#270)
M6Web/m6web.github.io: _posts/2021-07-16-bedrock-dev-facts-14.md

---
layout: post
title: "Bedrock Dev Facts #14"
description: "Les plus belles phrases de nos équipes techniques"
author:
  name: Bedrock
  avatar:         
  email:          
  twitter: Bedrock_Tech      
  facebook:       
  github:    
category: 
tags: [devfacts, humour]
image:
  feature: posts/devfacts.jpg
  credit: Game Of Thrones
  creditlink: 
comments: true  
---

Apportez vos [devfacts](/tags.html#devfacts) à la plage cet été!

# Les phases de test
> Pour une fois qu'on est payé à faire des 500 

# Le panier de crabes
> _ On attend le GoNoGo
> 
> _ On ne pourrait pas attendre le RustNoRust ce serait plus performant

# 1... 2... 3... extincteurs

> > Parlant au mec d'AWS

Reviewer: AWS c'est en 3 lettres aussi, et même si ce n'est pas français ça peut porter à confusion…
J'sais pas comment le rendre plus explicite, j'y réfléchis

> > Parlant au mec d'AWS

Reviewer: WYT ?

```suggestion
> Parlant à un hébergeur cloud
```

>
> "Tu sais la petite boite française en trois lettres qui a eu chaud ce matin?"

# Échec et mat
> We’ve been hacked by our incompetence

# Le vrai du faux
> C'est pas le bon true

# Darladidadada

> <blockquote class="twitter-tweet"><p lang="fr" dir="ltr">Quand tu fais du ménage dans ton compte AWS de tests et que tu te dis qu&#39;un collègue doit avoir l&#39;oreille musicale 🎶 <a href="https://t.co/yIINlboQBy">pic.twitter.com/yIINlboQBy</a></p>&mdash; Pascal MARTIN (@pascal_martin) <a href="https://twitter.com/pascal_martin/status/1370331336477659137?ref_src=twsrc%5Etfw">March 12, 2021</a></blockquote> 

Reviewer 2: dommage qu'on ne voit pas l'image dans l'article ^^

Author: L'intégration twitter est absente sur le blog malheureusement

# Review in progress

> ![screenshot d'une Pull request avec à la fois in review et in progress en label](/images/posts/dev-facts-14/in-porgress-review.png)

Reviewer 2: typo dans le nom du fichier ? image non affichée dans l'article

> ![screenshot d'une Pull request avec à la fois in review et in progress en label](/images/posts/dev-facts-14/in-porgress-review.png)

Reviewer 3: ```suggestion
![screenshot d'une Pull request avec à la fois in review et in progress en label](/images/posts/dev-facts-14/in-progress-review.png)
```

# IPV6 c'est pas encore hyper clair
> Oui mais c’est une grosse IP

# La confiance
> Normalement, si tout va bien, ça marche!

# Alors cette mise en production ? 
> Elle s'est bien passée mais il a fallu revert

# Et oui mon petit
> L’Internet, c’était nous

# Une API c'est pas si mal en fait

> > Quand je vois l'UX de la console AWS, je me demande si il y a pas un véritable volonté à nous pousser à faire de _l'infrastructure as code_

Reviewer 4: ```suggestion
> Quand je vois l'UX de la console AWS, je me demande s'il n'y a pas une véritable volonté à nous pousser à faire de _l'infrastructure as code_
```

(je propose mais si tu préfères retranscrire exactement sans y toucher, ça se tient aussi)

# De la part d'un dev backend Php 😱
> J'aime mettre du javascript partout

# Le retour du come-back de l’automatisation manuelle
> On pourra faire un rattrapage à moitié manuel, mais quand même automatique

# Dans les méandres des fonctionnalités de Airtable
> Bon, on va faire simple, on va écrire du JS

# Un mercredi à 15h55
> Cette phrase est trop complexe pour un lundi matin

# À bicyclette 🚲
> Le client il s’en fout que tu mettes des petites roues sur ton vélo, il veut juste que t’avances sans te péter la gueule

# Entendu dans une réunion à distance
> Rassurez-vous c'est pas moi qui ronfle c'est mon chat

# En parlant de choix techniques
> Attention le choix pris aujourd’hui fera jurisprudence sur la suite 

# C'est géré ! 
> Je crois que l’on gère cette erreur, même si on ne la comprend pas

# 1 + 1 = 11
> 30 est un nombre énorme

# Entendu juste après une grosse MEP
> Oh! Il faut que je mette en place un mail de vacances.

> 
> # En parlant de masturbation intellectuelle
> > La masturbation, c’est pour se faire du bien

Author: ```suggestion
```

# Des frites 🍟
> Mes critères de succès, c’est les grosses patates

# PO, pas un métier facile
> Le dev: "Est-ce que les PO peuvent le faire ?"
> 
> Le PO: "On ne sait rien faire nous"

> # Bouger pas

Reviewer 5: ```suggestion
# Bougez pas
```
❓

> Je déploie le deployer

_Une légende raconte qu'il faut déployer deux fois le deployer_

# Une petite équipe
> On était 80 stagiaires sur le projet

# 🎰

> > La magie du chmod -R 777. Je te dirais pas que j'en ai réjouté dans mes 3 dernières PR pour fixer de la CI

Reviewer 6: ```suggestion
> La magie du chmod -R 777. Je te dirais pas que j'en ai rajouté dans mes 3 dernières PR pour fixer de la CI
```
